apps/web/src/components/editor/tiptap.tsx

```diff
@@ -460,7 +460,7 @@ function TiptapWrapper(
             : EDITOR_ZOOM.STEP;
         const zoom = Math.min(
           EDITOR_ZOOM.MAX,
-          Math.max(EDITOR_ZOOM.MIN, editorConfig.zoom + delta)
+          Math.max(EDITOR_ZOOM.MIN, Math.round(editorConfig.zoom + delta))
```

Comment:
I'm including the `editorConfig.zoom` value in `Math.round` instead of just `delta`, because users who have faced this bug, might not be able to fix zoom value unless they clear local storage. Please let me know if this is okay.

Reply:
Good idea.